Jenkinsfile.integration

```diff
@@ -202,6 +204,39 @@ pipeline {
                 sh "./run.sh setup_airship"
             }
         }
+
+        stage('Run Tempest') {
+            options {
+                timeout(time: 240, unit: 'MINUTES', activity: true)
+            }
+            when {
+                allOf {
+                    expression { env.run_tempest == 'true' }
+                    expression { params.deployment == 'airship' }
+                    expression { return  TestFunctional }
+                }
+            }
+            steps {
+                sh """
+                  export TEMPEST_TEST_TYPE=${env.tempest_test_type}
+                  ./run.sh test
+                """
+            }
+            post {
+                always {
+                    steps {
```

Comment:
isn't that "script" instead of steps?

See jenkins log.

Reply:
Thanks for noticing it. Looks like got confused with wording of syntax https://jenkins.io/doc/book/pipeline/syntax/#post 

Made a change and see if it works.

Reply:
Ended up implementing tempest logs in ansible play so this section is no longer needed and removed from jenkins file.